audit/audit_syslog_storage_helper.cc

```diff
@@ -108,7 +108,7 @@ future<> audit_syslog_storage_helper::write(const audit_info* audit_info,
     auto now = std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
     tm time;
     localtime_r(&now, &time);
-    sstring msg = seastar::format(R"(<{}>{:%h %e %T} scylla-audit: node="{}" category="{}" cl="{}" error="{}" keyspace="{}" query="{}" client_ip="{}" table="{}" username="{}")",
+    sstring msg = seastar::format(R"(<{}>{:%h %e %T} scylla-audit: node="{}", category="{}", cl="{}", error="{}", keyspace="{}", query="{}", client_ip="{}", table="{}", username="{}")",
```

Comment:
Not strictly related to the issue that this PR fixes, but - do we need to escape some characters for the field values? For example - what if the query contains a double quote sign? Is the syslog format documented anywhere, and if so - where can I find it?

Reply:
Ok, I see that the function `json_escape` is applied to the query string before it is printed.